src/Operation/ModifyCollection.php

```diff
@@ -33,9 +34,21 @@
  */
 class ModifyCollection implements Executable
 {
+    /**
+     * @var string
+     */
```

Comment:
These type annotations for private members should be removed. These shouldn't appear in generated documentation and userland code should not interact with them.